core/src/main/java/com/linecorp/armeria/client/Endpoint.java

```diff
@@ -433,6 +438,96 @@ public String authority() {
         return this.authority = authority;
     }
 
+    /**
+     * Converts this endpoint into an URI using the {@code scheme}.
```

Comment:
nit: an URI -> a URI
The article depends on the pronunciation of the following word.

Reply:
I apply it to other comments too.
Thanks!!